package.json

```diff
@@ -1,46 +1,41 @@
 {
   "name": "chokidar",
   "description": "Minimal and efficient cross-platform file watching library",
-  "version": "3.5.2",
+  "version": "4.0.0-alpha",
   "homepage": "https://github.com/paulmillr/chokidar",
   "author": "Paul Miller (https://paulmillr.com)",
   "contributors": [
     "Paul Miller (https://paulmillr.com)",
     "Elan Shanker"
   ],
   "engines": {
-    "node": ">= 8.10.0"
+    "node": ">= 14.10.0"
   },
-  "main": "index.js",
+  "type": "module",
+  "main": "lib/index.js",
   "dependencies": {
-    "anymatch": "~3.1.2",
-    "braces": "~3.0.2",
-    "glob-parent": "~5.1.2",
-    "is-binary-path": "~2.1.0",
-    "is-glob": "~4.0.1",
-    "normalize-path": "~3.0.0",
-    "readdirp": "~3.6.0"
+    "is-binary-path": "2.1.0",
+    "normalize-path": "3.0.0",
+    "readdirp": "github:paulmillr/readdirp"
```

Comment:
This reduction in dependencies is awesome! 😍 💯 

Are the versions pinned on purpose?

Reply:
yes, to ensure packages don't get hacked and replaced by malware